src/components/LoadedCardsDisplayer.js

```diff
@@ -7,12 +8,16 @@ const LoadedCardsDisplayer = (props) => {
       <Row className="d-flex flex-wrap">
         {props.loadedCards.map((card) => (
           <Col key={card.id} xs={4} md={3} className="p-3">
-            <img
-              src={card.image_uris === undefined ? "" : card.image_uris.normal}
-              className="img-fluid zoom"
-              alt="Card"
-            ></img>
-            <p style={{ textAlign: "center" }}>{card.name}</p>
+            <Link to={`/${card.name}/${card.id}`}>
+              <img
+                src={
+                  card.image_uris === undefined ? "" : card.image_uris.normal
+                }
+                className="img-fluid zoom"
+                alt="Card"
+              ></img>
+            </Link>
+            <h5 style={{ textAlign: "center" }}>{card.name}</h5>
```

Comment:
I would recommend to use Bootstrap's `.text-center` instead ([click for more info](https://getbootstrap.com/docs/4.5/utilities/text/#text-alignment)).